nncf/quantization/algorithms/smooth_quant/algorithm.py

```diff
@@ -63,6 +63,10 @@ def __init__(self, subset_size: int = 300, inplace_statistics: bool = True, alph
         :param alpha: The parameter that regulates the calculation of the scale.
             The default value is 0.95. Negative value switches off the algorithm.
         """
+
+        if alpha < 0:
+            raise RuntimeError("Smooth Quant algorithm does not support negative alpha parameter!")
```

Comment:
I don't know, maybe should we keep `if self._alpha < 0:` conditions or do not add `RuntimeError`. @alexsu52, what do you think?

Reply:
LGTM. @AlexKoff88, do you have a comment?

Reply:
I am ok with this too.